README.md

```diff
@@ -11,7 +11,7 @@ Demo here: https://nadbm.github.io/react-datasheet/
 Examples are located in https://github.com/nadbm/react-datasheet/tree/master/docs/src/examples
 
 Current features:
-* Select cells, copy-paste cells
+* Select cells, cut, copy and paste cells
```

Comment:
Mentioned "cut" in "Current features", not sure what else needs to be added, since this is a very small feature and no API was changed

Reply:
@nadbm, should I anything else?

Reply:
@evgenykochetkov no that looks good. I will merge it soon